class-pikpay.php

```diff
@@ -1066,118 +1057,118 @@ public function payment_fields()
                 $price_increase_message .= "<span id='price-increase-12' class='price-increase-message' style='display: none; color: red;'> " . $lang["PAYMENT_INCREASE"] . " " . $this->price_increase_12 . "% = " . $amount12 . "</span>";
             } else {
                 $amount12 = $order_total;
                 $price_increase_message .= "<span id='price-increase-12' class='price-increase-message' style='display: none; color: red;'></span>";
             }
 
             if ($this->price_increase_13 != 0) {
-                $amount13 = $order_total + ($order_total * $this->price_increase_13 / 100);
+                $amount13 = number_format(($order_total + ($order_total * $this->price_increase_13 / 100) / 100), '.', ' ');
                 $price_increase_message .= "<span id='price-increase-13' class='price-increase-message' style='display: none; color: red;'> " . $lang["PAYMENT_INCREASE"] . " " . $this->price_increase_13 . "% = " . $amount13 . "</span>";
             } else {
                 $amount13 = $order_total;
                 $price_increase_message .= "<span id='price-increase-13' class='price-increase-message' style='display: none; color: red;'></span>";
             }
 
             if ($this->price_increase_14 != 0) {
-                $amount14 = $order_total + ($order_total * $this->price_increase_14 / 100);
+                $amount14 = number_format(($order_total + ($order_total * $this->price_increase_14 / 100) / 100), '.', ' ');
                 $price_increase_message .= "<span id='price-increase-14' class='price-increase-message' style='display: none; color: red;'> " . $lang["PAYMENT_INCREASE"] . " " . $this->price_increase_14 . "% = " . $amount14 . "</span>";
             } else {
                 $amount14 = $order_total;
                 $price_increase_message .= "<span id='price-increase-14' class='price-increase-message' style='display: none; color: red;'></span>";
             }
 
             if ($this->price_increase_15 != 0) {
-                $amount15 = $order_total + ($order_total * $this->price_increase_15 / 100);
+                $amount15 = number_format(($order_total + ($order_total * $this->price_increase_15 / 100) / 100), '.', ' ');
                 $price_increase_message .= "<span id='price-increase-15' class='price-increase-message' style='display: none; color: red;'> " . $lang["PAYMENT_INCREASE"] . " " . $this->price_increase_15 . "% = " . $amount15 . "</span>";
             } else {
                 $amount15 = $order_total;
                 $price_increase_message .= "<span id='price-increase-15' class='price-increase-message' style='display: none; color: red;'></span>";
             }
 
             if ($this->price_increase_16 != 0) {
-                $amount16 = $order_total + ($order_total * $this->price_increase_16 / 100);
+                $amount16 = number_format(($order_total + ($order_total * $this->price_increase_16 / 100) / 100), '.', ' ');
```

Comment:
@KipepeoYaMachungwa we should move this:
```php
number_format(($order_total + ($order_total * $this->price_increase_16 / 100) / 100), '.', ' ');
```
to a function, eg:
```
function format_amount($total, $price_increase) {
return number_format(($total + ($total * $price_increase / 100) / 100), '.', ' ');
}
```

Additionally PHP's number_format docs states:
```php
number_format(
    float $num,
    int $decimals = 0,
    ?string $decimal_separator = ".",
    ?string $thousands_separator = ","
): string

```
Execution of this code:
```php
echo number_format(1000, '.', ' ');
```
Results in:
```
Fatal error: Uncaught TypeError: number_format(): Argument #2 ($decimals) must be of type int, string given in /home/user/scripts/code.php:3
Stack trace:
#0 /home/user/scripts/code.php(3): number_format(1000, '.', ' ')
#1 {main}
  thrown in /home/user/scripts/code.php on line 3
```

Are we sure this is working as expected?